<ui version="4.0" >
 <author>Marcos Mezo</author>
 <class>ClienteBase</class>
 <widget class="QWidget" name="ClienteBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>629</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Cliente</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_clientes.48x48.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>331</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar" >
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar" >
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="mui_tab" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Tab1" >
      <attribute name="title" >
       <string>&amp;Datos generales</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="14" column="1" colspan="7" >
        <widget class="QLineEdit" name="mui_bancocliente" />
       </item>
       <item row="14" column="0" >
        <widget class="QLabel" name="textLabel2_4_3" >
         <property name="text" >
          <string>Datos bancarios:</string>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="4" >
        <widget class="QLabel" name="textLabel2_4_2_2" >
         <property name="text" >
          <string>Comentarios:</string>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="4" >
        <widget class="QTextEdit" name="mui_comentcliente" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="12" column="4" colspan="4" >
        <widget class="QLabel" name="textLabel2_4_2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>32766</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text" >
          <string>Datos de comercio electronico:</string>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="4" colspan="4" >
        <widget class="QTextEdit" name="mui_clientEcomerceData" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="9" column="6" colspan="2" >
        <widget class="QCheckBox" name="mui_recargoeqcliente" >
         <property name="text" >
          <string>Aplicar Recargo de Equivalencia</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6" colspan="2" >
        <widget class="QLineEdit" name="mui_mailcliente" >
         <property name="maxLength" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="7" column="6" colspan="2" >
        <widget class="QLineEdit" name="mui_movilcliente" >
         <property name="maxLength" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="6" column="6" colspan="2" >
        <widget class="QLineEdit" name="mui_faxcliente" >
         <property name="maxLength" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="6" >
        <widget class="QLabel" name="textLabel2_7" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Codigo postal:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="7" >
        <widget class="QLineEdit" name="mui_cpcliente" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="6" colspan="2" >
        <widget class="BusquedaProvincia" native="1" name="mui_idprovincia" >
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <widget class="QLineEdit" name="mui_cifcliente" >
         <property name="maxLength" >
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_8_2" >
         <property name="text" >
          <string>Pagina web:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="6" >
        <widget class="QLineEdit" name="mui_nomaltcliente" >
         <property name="maxLength" >
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_5" >
         <property name="text" >
          <string>C.I.F./N.I.F./N.I.E.:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="6" >
        <widget class="QLineEdit" name="mui_corpcliente" >
         <property name="maxLength" >
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="6" column="5" >
        <widget class="QLabel" name="textLabel2_8_4" >
         <property name="text" >
          <string>Numero de fax:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="5" >
        <widget class="QLabel" name="textLabel2_8_5" >
         <property name="text" >
          <string>Telefono movil:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_4" >
         <property name="text" >
          <string>Nombre comercial:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="3" >
        <widget class="QLineEdit" name="mui_urlcliente" >
         <property name="maxLength" >
          <number>150</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_9" >
         <property name="text" >
          <string>Poblacion:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="6" colspan="2" >
        <widget class="QLineEdit" name="mui_codcliente" >
         <property name="maxLength" >
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_11" >
         <property name="text" >
          <string>Telefono del trabajo:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_2" >
         <property name="text" >
          <string>Nombre completo:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="3" >
        <widget class="QLineEdit" name="mui_telcliente" >
         <property name="maxLength" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="8" column="5" >
        <widget class="QLabel" name="textLabel2_8_3" >
         <property name="text" >
          <string>Correo electronico:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="3" >
        <widget class="QLineEdit" name="mui_poblcliente" >
         <property name="maxLength" >
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_2_2" >
         <property name="text" >
          <string>Empresa:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="5" >
        <widget class="QLabel" name="textLabel2_10" >
         <property name="text" >
          <string>Codigo de cliente:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="5" >
        <widget class="QLabel" name="textLabel2_9_2" >
         <property name="text" >
          <string>Provincia:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="3" >
        <widget class="QLineEdit" name="mui_teltrabcliente" >
         <property name="maxLength" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_6" >
         <property name="text" >
          <string>Direccion:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_8" >
         <property name="text" >
          <string>Telefono fijo:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="6" >
        <widget class="QLineEdit" name="mui_nomcliente" >
         <property name="maxLength" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4" >
        <widget class="QLineEdit" name="mui_dircliente" >
         <property name="maxLength" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_8_6" >
         <property name="text" >
          <string>Forma de pago:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="3" >
        <widget class="BusquedaFormaPago" name="mui_idforma_pago" />
       </item>
       <item row="10" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_8_7" >
         <property name="text" >
          <string>Regimen fiscal:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="3" >
        <widget class="BusquedaRegimenFiscal" name="mui_regimenfiscalcliente" />
       </item>
       <item row="11" column="0" colspan="8" >
        <widget class="QFrame" name="m_frameplugin" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab2" >
      <attribute name="title" >
       <string>Pres&amp;upuestos</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="PresupuestoList" native="1" name="m_listpresupuestos" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab3" >
      <attribute name="title" >
       <string>Pedidos</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="PedidosClienteList" native="1" name="m_listpedidos" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab4" >
      <attribute name="title" >
       <string>Albaranes</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="AlbaranClienteList" native="1" name="m_listalbaranes" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab5" >
      <attribute name="title" >
       <string>&amp;Facturas</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="FacturasList" native="1" name="m_listfacturas" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab6" >
      <attribute name="title" >
       <string>Cobros</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="0" column="0" >
        <widget class="CobrosList" name="m_listcobros" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_clientes.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar cliente</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar cliente</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Images/edit-save.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_eliminar" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar cliente</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar cliente</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:Images/edit-delete.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_informe" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir informe</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir informe</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>371</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5" />
 <customwidgets>
  <customwidget>
   <class>PedidosClienteList</class>
   <extends>QWidget</extends>
   <header>pedidosclientelist.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaProvincia</class>
   <extends>QWidget</extends>
   <header>busquedaprovincia.h</header>
  </customwidget>
  <customwidget>
   <class>PresupuestoList</class>
   <extends>QWidget</extends>
   <header>presupuestolist.h</header>
  </customwidget>
  <customwidget>
   <class>AlbaranClienteList</class>
   <extends>QWidget</extends>
   <header>albaranclientelist.h</header>
  </customwidget>
  <customwidget>
   <class>FacturasList</class>
   <extends>QWidget</extends>
   <header>facturaslist.h</header>
  </customwidget>
  <customwidget>
   <class>CobrosList</class>
   <extends>QTableWidget</extends>
   <header>cobroslist.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaRegimenFiscal</class>
   <extends>QComboBox</extends>
   <header>busquedaregimenfiscal.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaFormaPago</class>
   <extends>QComboBox</extends>
   <header>busquedaformapago.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_tab</tabstop>
  <tabstop>mui_cifcliente</tabstop>
  <tabstop>mui_codcliente</tabstop>
  <tabstop>mui_nomcliente</tabstop>
  <tabstop>mui_corpcliente</tabstop>
  <tabstop>mui_nomaltcliente</tabstop>
  <tabstop>mui_dircliente</tabstop>
  <tabstop>mui_cpcliente</tabstop>
  <tabstop>mui_poblcliente</tabstop>
  <tabstop>mui_telcliente</tabstop>
  <tabstop>mui_faxcliente</tabstop>
  <tabstop>mui_teltrabcliente</tabstop>
  <tabstop>mui_movilcliente</tabstop>
  <tabstop>mui_urlcliente</tabstop>
  <tabstop>mui_mailcliente</tabstop>
  <tabstop>mui_comentcliente</tabstop>
  <tabstop>mui_clientEcomerceData</tabstop>
  <tabstop>mui_bancocliente</tabstop>
  <tabstop>mui_guardar</tabstop>
  <tabstop>mui_eliminar</tabstop>
  <tabstop>mui_informe</tabstop>
  <tabstop>mui_cancelar</tabstop>
  <tabstop>mui_aceptar</tabstop>
 </tabstops>
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
